chef_master/source/resource_remote_file.rst

```diff
@@ -600,6 +619,58 @@ To prevent the chef-client from re-downloading files that are already present on
 
 The desired approach just depends on the desired workflow. For example, if a node requires a new file every day, using the checksum approach would require that the local checksum be updated and/or verified every day as well, in order to ensure that the local checksum was the correct one. Using a conditional ``GET`` in this scenario will greatly simplify the management required to ensure files are being updated accurately.
 
+Access a remote UNC path on Windows
+-----------------------------------------------------
+The ``remote_file`` resource now supports the use of credentials on Windows when accessing a remote UNC path on Windows such as ``\\myserver\myshare\mydirectory\myfile.txt``. This
+allows access to the file at that path location even if the Chef client process identity does not have permission to access the file. The new properties ``remote_user``, ``remote_domain``, and ``remote_password`` may be used to specify credentials with access to the remote file so that it may be read. Please check the details of the usage of these properties above.
+
+**Note**: This feature is mainly used for accessing files between two nodes in different domains and having different user accounts.
+In case the two nodes are in same domain, ``remote_file`` resource does not need ``remote_user`` and ``remote_password`` specified because the user has the same access on both systems through the domain.
+
+For example:
+
+Accessing file from a (different) domain account
+
+.. code-block:: ruby
+
+   remote_file "E://domain_test.txt"  do
+     source  "\\\\myserver\\myshare\\mydirectory\\myfile.txt"
+     remote_domain "domain"
+     remote_user "username"
+     remote_password "password"
+   end
+
+OR
+
+.. code-block:: ruby
+
+   remote_file "E://domain_test.txt"  do
+     source  "\\\\myserver\\myshare\\mydirectory\\myfile.txt"
+     remote_user "domain\\username"
+     remote_password "password"
+   end
+
+Accessing file using a local account on the remote machine
+
+.. code-block:: ruby
+
+   remote_file "E://domain_test.txt"  do
+     source  "\\\\myserver\\myshare\\mydirectory\\myfile.txt"
+     remote_domain "."
```

Comment:
we might want to add a note above that the `.` is required to specify the local user account. That could be missed here if this goes unexplained.

Like under the "remote_domain" property section something like "If the remote system is domain joined, but it is necessary to authenticate as a local user on the remote system, set the domain to `.`, e:g: ` remote_domain "."`.